packages/terra-base/package.json

```diff
@@ -28,8 +28,7 @@
     "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "react": "^15.4.2"
```

Comment:
Here is the issue I am seeing:
1) lock react at 15.4.2 in the root directory and in terra-site.
2) `npm run clean:install` 
3) Inspect node_modules in any package that has a dependency on react. You will see a react directory with react 15.5.x.  This gets confusing quickly because I am unsure which version of react each of these packages is using during testing / runtime.

We can discuss more tomorrow.